recipes/configs/llama2/7B_lora_single_device.yaml

```
@@ -80,9 +80,29 @@ device: cuda
 dtype: bf16
 enable_activation_checkpointing: True
 
+# Show case the usage of pytorch profiler
+# Set enabled to False as it's only needed for debugging training
 # Show case the usage of pytorch profiler
 # Set enabled to False as it's only needed for debugging training
 profiler:
-  _component_: torchtune.utils.profiler
   enabled: False
-  output_dir: ${output_dir}/torchtune_perf_tracing.json
+
+  #Output directory of trace artifacts
+  output_dir: /tmp/profiling_outputs
```

Would be great to reuse the output_dir we are already creating above so we don't make too many stray folders
```suggestion
  output_dir: ${output_dir}/profiling_outputs
```

---

Done